che-launcher/launcher_cmds.sh

```diff
@@ -105,25 +106,34 @@ print_debug_info() {
   info "DOCKER_DAEMON_VERSION     = $(get_docker_daemon_version)"
   info ""
   info ""
-  info "--------- CHE INSTANCE INFO  ----------" 
-  info "CHE CONTAINER EXISTS      = $(che_container_exist && echo "YES" || echo "NO")"
-  info "CHE CONTAINER STATUS      = $(che_container_is_running && echo "running" || echo "stopped")"
-  if che_container_is_running; then
-    info "CHE SERVER STATUS         = $(server_is_booted && echo "running & api reachable" || echo "stopped")"
-    info "CHE VERSION               = $(get_server_version)"
-    info "CHE IMAGE                 = $(get_che_container_image_name)"
-    info "CHE SERVER CONTAINER ID   = $(get_che_server_container_id)"
-    info "CHE CONF FOLDER           = $(get_che_container_conf_folder)"
-    info "CHE DATA FOLDER           = $(get_che_container_data_folder)"
-    CURRENT_CHE_PORT=$(docker inspect --format='{{ (index (index .NetworkSettings.Ports "8080/tcp") 0).HostPort }}' ${CHE_SERVER_CONTAINER_NAME})
-    info "CHE USE URL               = http://${CHE_HOST_IP}:${CURRENT_CHE_PORT}"  
-    info "CHE API URL               = http://${CHE_HOST_IP}:${CURRENT_CHE_PORT}/swagger"
-    if has_debug; then
-      CURRENT_CHE_DEBUG_PORT=$(docker inspect --format='{{ (index (index .NetworkSettings.Ports "8000/tcp") 0).HostPort }}' ${CHE_SERVER_CONTAINER_NAME})
-      info "CHE JPDA DEBUG URL      = http://${CHE_HOST_IP}:${CURRENT_CHE_DEBUG_PORT}"  
+  info "--------- CHE INSTANCE LIST  ----------"
+  CURRENT_CHE_INSTANCES=$(docker ps -aq --filter "ancestor=${CHE_SERVER_IMAGE_NAME}:${CHE_VERSION}")
```

Comment:
Unfortunately `${CHE_VERSION}` is required here. The ancestor command did not return any results if you just provided `${CHE_SERVER_IMAGE_NAME}`.